### DISCUSSION TITLE
Release v1.2.0

### DISCUSSION THREAD

**Discussion by ChenglongMa** [ChenglongMa/LOLauncher#11] [General]
# 更新日志

1. 添加支持PBE测试服
2. 全新UI设计
3. 支持暗黑模式

## What's Changed
* Dev 1.2.0 pbe support by @ChenglongMa in https://github.com/ChenglongMa/LOLauncher/pull/10

**Full Changelog**: https://github.com/ChenglongMa/LOLauncher/compare/1.1.0...1.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ChenglongMa/LOLauncher/releases/tag/1.2.0'>Release v1.2.0</a>.</em>

**Comment by Ptou17** [comment]
厉害大佬！！！！支持！！！

**Reply by ChenglongMa** [reply]
多谢鼓励！！😊😊

**Comment by innkbamboo** [comment]
支持+1
